projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Observable, of as observableOf, BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class GoSideNavService {
+  public navState = true;
```

Comment:
Do you think we should change this to be a bit more specific as to what this is? Maybe `navOpen`?